src/Middleware/CakeSentryPerformanceMiddleware.php

```diff
@@ -76,6 +77,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         $this->addQueryData();
         $listener = new EventListener();
         EventManager::instance()->on($listener);
+        EventManager::instance()->on(new HttpEventListener());
```

Comment:
In the next major realease of this plugin I would like to unify the listener classes into their own folder/namespace. Didn't expect that I already need multiple implementations of different event listeners already.